src/commands/create-parts.ts

```diff
@@ -51,6 +55,10 @@ export default class CreateParts extends BaseCommand {
   };
 
   public async run(): Promise<void> {
+    await this.innerRun(createPartFromFileIfNotExists);
+  }
+
+  public async innerRun(createPartsFn: CreatePartsFn): Promise<void> {
```

Comment:
Maybe there's a better way to do this, I know there is with `jest`. But with `sinon` and `fancy-test`, this is the the only way I was able to get a stub injected.